### PR TITLE
Updated human format

### PR DESCRIPTION
**Pull request by jennymahmoudi**
Updated human format for tasks, commands, scripts and files

### CODE REVIEW

File: ux/bolt-CLI-spec.md [hunk 1]
```diff
@@ -86,47 +86,72 @@ PARAMETERS:
...
 Duration: 27 sec
 ~~~
 
+
 - colors should match puppet job run output for started, finished, and errors.
 - For tasks, capitalization of the finished node results comes from module (eg. status vs. Status).
-- status will be succeeded or failed for command and scripts; task status may be something else specified by task.
 - STDOUT or STDERR labels will be printed only if the outout from that stream is non-empty.
-
+- file will only output STDERR; there will be no STDOUT
```
Comment (adreyer) [inline ux/bolt-CLI-spec.md]: There is not STDERR just

Finished on node-3 (in green)
or
Failed on node-3 (in red)
  Could not connect to node-3 (in red)

Comment (jennymahmoudi) [reply ux/bolt-CLI-spec.md]: Is the error message _error?  Is the file output a special case of tasks, commands, or should it have its own spec section?

**Comment by jennymahmoudi** [comment]
@adreyer made the changes. ready for review/merge

File: ux/bolt-CLI-spec.md [hunk 2]
```diff
@@ -86,49 +86,96 @@ PARAMETERS:
...
-  status: [succeeded/failed/<task-specific staus>]
-    STDOUT:
-      [stdout output...]
-    STDERR:
-      [stderr output...] 
+[Succeeded/Failed] on node-1
+  STDOUT:
+    [stdout output...]
+  STDERR:
+    [stderr output...]
```
Comment (MikaelSmith) [inline ux/bolt-CLI-spec.md]: Should we print
```
STDERR:

```
when there's no stderr output?

Comment (jennymahmoudi) [reply ux/bolt-CLI-spec.md]: No. There is a note in the spec: "STDOUT or STDERR labels will be printed only if the output from that stream is non-empty."

Comment (adreyer) [reply ux/bolt-CLI-spec.md]: oops I didn't implement that.
Should any message be printed if there is no output?

Comment (jennymahmoudi) [reply ux/bolt-CLI-spec.md]: Let's go with no message for now, unless we learn of a use case where one is required. The status will be printed regardless.

**Pull request merged**